java/src/main/java/com/cybersource/ws/client/Connection.java

```diff
@@ -126,6 +126,8 @@ private void checkForFault()
             logger.log(Logger.LT_INFO, "Reading response...");
 
             int responseCode = getHttpResponseCode();
+            
+            logResponseHeaders();
```

Comment:
remove this line. why this is require here, you have already kept those two methods below

Reply:
removed extra lines